subcommands/waves/sign.go

```diff
@@ -0,0 +1,43 @@
+package waves
+
+import (
+	"github.com/docker/go/canonical/json"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/subcommands"
+	tuf "github.com/theupdateframework/notary/tuf/data"
+)
+
+func init() {
+	signCmd := &cobra.Command{
+		Use:   "sign <wave>",
+		Short: "Sign an existing wave targets with additional key",
```

Comment:
Is it `key` or could be `keys`?

Reply:
It is a KEY (no _s_).
See the other comment below explaining why.